docs/zh/quickstart/beginner_must_read.md

```diff
@@ -42,15 +62,49 @@ docker创建OpenMLDB见[快速上手](./openmldb_quickstart.md)，请注意文
 csv文件格式有诸多不便，更推荐使用parquet格式，需要OpenMLDB集群版并启动taskmanager组件。
 ```
 
-## SQL限制
+## SQL
 
 OpenMLDB并不完全兼容标准SQL。所以，部分SQL执行会得不到预期结果。如果发现SQL执行不符合预期，请先查看下SQL是否满足[功能边界](./function_boundary.md)。
 
+### SQL编写指南
+
+首先，OpenMLDB SQL 通常是使用`WINDOW`（包括`WINDOW UNION`），`LAST JOIN`子句，它们能保证在任何模式下使用。可以跟随教程"基于 SQL 的特征开发"[(上)](../tutorial/tutorial_sql_1.md)[(下)](../tutorial/tutorial_sql_2.md)进行学习。
+
+如果使用`WHERE`，`WITH`，`HAVING`等子句，需要注意限制条件。在每个子句的详细文档中都有具体的说明，比如[`HAVING`子句](../openmldb_sql/dql/HAVING_CLAUSE.md)在在线请求模式中不支持。翻阅OpenMLDB SQL的DQL目录，或使用搜索功能，可以快速找到子句的详细文档。
+
+在不熟悉OpenMLDB SQL的情况下，我们建议从下到上编写SQL，确保每个子句都能通过，再逐步组合成完整的SQL。
+
+推荐使用[OpenMLDBSimulator](https://github.com/vagetablechicken/OpenMLDBSimulator)进行SQL探索，SQL完成后再去真实集群进行上线。Simulator可以不依赖真实OpenMLDB集群，在一个交互式虚拟环境中，快速创建表、校验SQL、导出当前环境等等，详情参考该项目的README。使用Simulator不需要操作集群，也就不需要测试后清理集群，还可通过少量的数据进行SQL运行测试，比较适合SQL探索时期。
```

Comment:
the full name can be "OpenMLDB SQL Emulator"
emulator should be the correct word

Reply:
done